Source/KakapoServer.swift

```diff
@@ -37,15 +39,29 @@ import Foundation
  ```
  */
 public final class KakapoServer: NSURLProtocol {
-    
-    private static var routers: [Router] = []
+
+    /**
+     The overall list of `Router` objects, which is know to all `KakapoServer` instances.
+     Visibility is on the `KakapoServer` class level, because `NSURLProtocol` just allows us to register
+     class elements (no instances are possible).
+     */
+    public static var routers: [Router] = []
+
+    /**
+     `true`, if the `request` of the `KakapoServer` instance has been cancelled, otherwise `false`.
+
+     Default: `false`
+
+     Note: calls to `stopLoading()` will set this value to `true`
+     */
+    public private(set) var requestCancelled:Bool = false
```

Comment:
@joanromano we need contributors and style guidelines... for this time we can just apply the style ourself (e.g. Space missinggn between : and Bool)